### PR TITLE
check repoclosure against the whole repo, not only the new package

### PR DESCRIPTION
**Pull request by evgeni**
also check dependant repos (plugins/katello for core)

For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -22,14 +22,18 @@ foreman_scl_packages:
       - koji-foreman
     nightly_releaser: koji-foreman-jenkins
     repoclosure_config: repoclosure/yum_el7.conf
+    repoclosure_target_repos:
+      - downloaded_rpms
```
Comment (ehelms) [inline package_manifest.yaml]: Previously this was 'hidden' as a nuance of how Obal handled scratch builds. This seems strange to have to surface here? Could Obal just include this as a default lookup location?

Comment (evgeni) [reply package_manifest.yaml]: Obal certainly could. We have configs (see below for repoclosure only examples) where we don't have a downloaded_rpms folder. What would happen if we still add it in that case. We could make obal only add it if present?

File: package_manifest.yaml [hunk 2]
```diff
@@ -200,13 +204,17 @@ foreman_nonscl_packages:
...
+      - el7-foreman-nightly
+      - el7-foreman-plugins-nightly
```
Comment (ekohl) [inline package_manifest.yaml]: Does this mean we no longer check repoclosure for nonscl without plugins present? Should we have a separate run for that so you never break repoclosure on nightly, but can break plugins and katello. For example when you're doing the fog migration.

File: package_manifest.yaml (hunk 1 above)
Comment (evgeni) [reply package_manifest.yaml]: Implemented in https://github.com/theforeman/obal/pull/186

File: package_manifest.yaml (hunk 2 above)
Comment (evgeni) [reply package_manifest.yaml]: This is a good point. My intention here was "did a change in nonscl break plugins?", but this obviously also makes plugins resolve the deps...

Why are we always finding more layers of complexity in this?

Anyways, this PR will be closed until we sort out https://github.com/theforeman/obal/pull/186

**Comment by evgeni** [comment]
closing until https://github.com/theforeman/obal/pull/186 is finished

**Pull request closed without merging**